discussion/app/views/fragments/commentShare.scala.html

```diff
@@ -0,0 +1,34 @@
+@(comment: Comment)
+@import org.jsoup.Jsoup
+@import org.jsoup.safety.Whitelist
+
+<div class="sharing-component d-comment__action">
+
+    <div class="sharing-text">@fragments.inlineSvg("share", "icon") Share</div>
+
+    <div class="block-share block-share--gallery sharing-buttons" data-link-name="block share">
+    @defining(s"https://gu.com${comment.discussion.key}#comment-${comment.id}") { permalink =>
+        @defining(Jsoup.clean(comment.body, Whitelist.none())) { commentBody =>
+            <a class="social__action social-icon-wrapper"
+            href="@ShareLinks.createShareLinkForComment(platform = Facebook, href = permalink, text = "", quote = Some(s"${comment.profile.displayName} commented: '$commentBody'")).href"
+            target="_blank"
+            data-link-name="social facebook">
+                <span class='inline-icon__fallback button share-modal__item share-modal__item--facebook'>Share on Facebook</span>
+                @fragments.inlineSvg("share-facebook", "icon", List("rounded-icon social-icon social-icon--facebook"))
+                <span class="u-h">Facebook</span>
+            </a>
+
+            <a class="social__action social-icon-wrapper"
+            href="@ShareLinks.createShareLinkForComment(platform = Twitter, href = comment.webUrl, text = s"'${commentBody.take(111)}...'").href"
+            target="_blank"
+            data-link-name="social twitter">
```

Comment:
We should try to associate this `data-link-name` to comment social: eg:  `social-comment : twitter`